test/integration/getserversideprops-preview/test/index.test.js

```diff
@@ -175,6 +175,18 @@ function runTests(startServer = nextStart) {
     )
   })
 
+  it('should fetch preview data on CST', async () => {
+    await browser.get(`http://localhost:${appPort}/to-index`)
```

Comment:
We still need to create a browser instance right?

Reply:
The browser is created above and re-used for tests.